### PR TITLE
K8SPS-63: add ps-operator

### PR DESCRIPTION
**Pull request by pooknull**
https://jira.percona.com/browse/K8SPS-63

### CODE REVIEW

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| api-tests | passed |

**Review by egegunes — APPROVED** [review]

**Review by nonemax — APPROVED** [review]

**Review by tplavcic — CHANGES_REQUESTED** [review]
We need to add some tests for PS operator cases.

**Review by tplavcic — APPROVED** [review]

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| api-tests | passed |

**Review by nonemax — DISMISSED** [review]

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| api-tests | failed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| api-tests | failed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| api-tests | passed |

File: api-tests/apply_route_test.go [hunk 1]
```diff
@@ -529,6 +568,44 @@ func TestApplyPGReturnedVersions(t *testing.T) {
...
+		{"latest", "0.2.0", "8.0.28-20"},
+
+		// test recommended
+		{"recommended", "0.2.0", "8.0.28-20"},
+
+		// test exact
+		{"8.0.27", "0.2.0", "8.0.27-18"},
+
+		//test with suffix
+		{"8-latest", "0.2.0", "8.0.28-20"},
```
Comment (tplavcic) [inline api-tests/apply_route_test.go:589]: In PXC we have `8.0-recommended` so I propose we have the same for consistency reasons and also we don't know what Oracle will do with MySQL version.
So let's have `8.0-recommended` and `8.0-latest`

**Review by tplavcic — CHANGES_REQUESTED** [review]

File: api-tests/apply_route_test.go (hunk 1 above)
Comment (pooknull) [reply api-tests/apply_route_test.go]: https://github.com/Percona-Lab/percona-version-service/pull/88/commits/6a986830e4a903b5d402b33cc3b5a5a0695648e7

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| api-tests | passed |

**Review by nonemax — APPROVED** [review]

**Review by tplavcic — APPROVED** [review]

**Pull request merged**